### PR TITLE
feat(FEC-10699): add Linear media type recognition by OTT BE response

### PR DESCRIPTION
**Pull request by RoyBregman**
### Description of the Changes

In OTT BE response, when KalturaLiveAsset and enableTrickPlay is true - set dvr to true

resolves FEC-10699

### CheckLists

- [ ] changes have been done against master branch, and PR does not conflict
- [ ] new unit / functional tests have been added (whenever applicable)
- [ ] test are passing in local environment
- [ ] Travis tests are passing (or test results are not worse than on master branch :))
- [ ] Docs have been updated

### CODE REVIEW

File: src/k-provider/ott/provider-parser.js [hunk 1]
```diff
@@ -23,7 +23,7 @@ const MediaTypeCombinations: {[mediaType: string]: Object} = {
     [KalturaPlaybackContext.Type.TRAILER]: () => ({type: MediaEntry.Type.VOD}),
     [KalturaPlaybackContext.Type.PLAYBACK]: mediaAssetData => {
       if (parseInt(mediaAssetData.externalIds) > 0 || mediaAssetData.objectType === LIVE_ASST_OBJECT_TYPE) {
-        return {type: MediaEntry.Type.LIVE, dvrStatus: 0};
+        return {type: MediaEntry.Type.LIVE, dvrStatus: mediaAssetData.enableTrickPlay ? 1 : 0};
```
Comment (OrenMe) [inline src/k-provider/ott/provider-parser.js:26]: the condition in the ticket discuss `enableTrickPlay` and `mediaAssetData.objectType === LIVE_ASST_OBJECT_TYPE` but not `mediaAssetData.externalIds`

Comment (RoyBregman) [reply src/k-provider/ott/provider-parser.js:28]: Correct. pushed a fix

Comment (OrenMe) [reply src/k-provider/ott/provider-parser.js:28]: Can we add a test?

Comment (RoyBregman) [reply src/k-provider/ott/provider-parser.js:28]: We can, OTT doesn't have at all provider parser spec. I think we should add. Just a matter of time :)

Comment (OrenMe) [reply src/k-provider/ott/provider-parser.js:28]: please discuss with @yairans to add unit tests coverage for this

**Review by OrenMe — APPROVED** [review]

File: src/k-provider/ott/provider-parser.js [hunk 2]
```diff
@@ -22,8 +22,10 @@ const MediaTypeCombinations: {[mediaType: string]: Object} = {
   [KalturaAsset.Type.MEDIA]: {
     [KalturaPlaybackContext.Type.TRAILER]: () => ({type: MediaEntry.Type.VOD}),
     [KalturaPlaybackContext.Type.PLAYBACK]: mediaAssetData => {
-      if (parseInt(mediaAssetData.externalIds) > 0 || mediaAssetData.objectType === LIVE_ASST_OBJECT_TYPE) {
+      if (parseInt(mediaAssetData.externalIds) > 0) {
         return {type: MediaEntry.Type.LIVE, dvrStatus: 0};
+      } else if (mediaAssetData.objectType === LIVE_ASST_OBJECT_TYPE) {
+        return {type: MediaEntry.Type.LIVE, dvrStatus: mediaAssetData.enableTrickPlay ? 1 : 0};
```
Comment (dan-ziv) [inline src/k-provider/ott/provider-parser.js:28]: @RoyBregman suggest to add enum for this (can't remember why we didn't do it).

**Review by dan-ziv — CHANGES_REQUESTED** [review]

File: src/k-provider/ott/provider-parser.js (hunk 2 above)
Comment (dan-ziv) [reply src/k-provider/ott/provider-parser.js:28]:
```js
DvrStatus.ON
DvrStatus.OFF
```

Comment (RoyBregman) [reply src/k-provider/ott/provider-parser.js:28]: done

Comment (dan-ziv) [reply src/k-provider/ott/provider-parser.js:28]: @RoyBregman but need also to assign the enum 😁

Comment (RoyBregman) [reply src/k-provider/ott/provider-parser.js:28]: Oh, i thought only declare it ;)
I pushed the wrong file... pushed it now

**Review by dan-ziv — DISMISSED** [review]

File: dist/playkit-ovp-provider.js [hunk 3]
```diff
@@ -1,2 +1,2 @@
...
...e.prototype;return t.add=function(e,t){var r=this,n=new e(t);if(n.isValid()){this._loaders.set(e.id,n);var i=this._multiRequest.requests.length,a=n.requests;this._multiRequest.retryConfig=this._networkRetryConfig,a.forEach((function(e){r._multiRequest.add(e)}));var s=Array.from(new Array(a.length),(function(e,t){return t+i}));this._loadersResponseMap.set(e.id,s)}},t.fetchData=function(){var e=this;return new Promise((function(t,r){e._multiRequest.execute().then((function(i){e._multiResponse=i.response,e.prepareData(i.response).success?t(e._loaders):r(new n.a(n.a.Severity.CRITICAL,n.a.Category.NETWORK,n.a.Code.API_RESPONSE_MISMATCH,{headers:i.headers}))}),(function(e){r(e)}))}))},t.prepareData=function(e){var t=this;return this._loaders.forEach((function(r,n){var i=t._loadersResponseMap.get(n);try{i&&i.length>0&&(r.response=e.results.slice(i[0],i[i.length-1]+1))}catch(e){return{success:!1,error:e}}})),{success:!0,data:this._loaders}},e}()},function(e,t,r){"use strict";r.d(t,"a",(function(){return c}));var n=r(3),i=r(1),a=r(5),s=r(0);function o(e){if(void 0===e)throw new ReferenceError("this hasn't been initialised - super() hasn't been called");return e}function u(e,t,r){return t in
```
Comment (OrenMe) [inline dist/playkit-ovp-provider.js:1]: do we need this file change?

Comment (RoyBregman) [reply dist/playkit-ovp-provider.js:1]: No, it was by mistake. I even pushed a rollback but it somehow still considered it changed...

Comment (RoyBregman) [reply dist/playkit-ovp-provider.js]: Reverted successfully

**Review by OrenMe — APPROVED** [review]

**Pull request merged**